content/en/01-about-pixie/03-pixie-ebpf.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Pixie + eBPF"
+metaTitle: "About Pixie | Pixie + eBPF"
+metaDescription: "How Pixie uses eBPF to automatically collect data"
+order: 3
+---
+
+Pixie uses [eBPF](https://www.brendangregg.com/ebpf.html) to drive much of its data collection.
+This eBPF-based approach enables Pixie to efficiently collect data in a way that requires no user instrumentation
+(i.e. no code changes, no redeployments).
+
+Pixie sets up eBPF probes to trigger on a number of kernel or user-space events,
+at which time it collects the data of interest, such as various protocol messages. Additional details on how
```

Comment:
Maybe: 
The next section will cover how Pixie uses eBPF in some of its key features.